api/src/main/java/org/openmrs/module/legacyui/GeneralUtils.java

```diff
@@ -68,4 +83,69 @@ public static Concept getConcept(String id) {
 		
 		return cpt;
 	}
+	
+	/**
```

Comment:
The transactional annotation won't have any impact here AFAIK, can you move this to the service you created?